src/app/containers/ImageWithPlaceholder/index.jsx

```diff
@@ -57,7 +57,10 @@ const ImageWithPlaceholder = ({
           />
         </Helmet>
       )}
-      <ImagePlaceholder css={isLoaded && `background: none`} ratio={ratio}>
+      <ImagePlaceholder
+        style={isLoaded ? { background: 'none' } : null}
```

Comment:
Do we normally use the [styled components](https://emotion.sh/docs/styled) approach to apply css to React components? Is this literally just an inline style?

I don't know if there are any downsides with use an inline style this way but I wonder why we've taken this approach here compared to other places; it seems the `css` prop wouldn't exist if `style` was the correct way to go?

Reply:
- `style` sets an inline style.
- `css` will add a style to the stylesheet and generate a class, which Emotion will then attach to this component.

The only difference is specificity, and for a style override like this both approaches work effectively.

Reply:
Just quoting @jroebu14 here:
> with inline styles you don't get media-queries, psuedo selectors, you have to use !important or write more inline styles to override, debugging UI bugs can become more difficult, inline styles can't be reused like class names can, can't be minified 

Seems we should consider using the css prop on that basis but with 2 approvals please go ahead with this PR if you wish, I have no major concerns.

Reply:
Simon is right that an inline style here is OK and seems harmless but I worry that we then open the gates to a wee display:block here, a wee !important there or nested selectors, or somebody new starts and begins doing the same because it's how we do things and it all gets out of hand!

Reply:
That's the reason for my reluctance here, doesn't seem very hard to be consistent with everywhere else in this case and will form the basis for someone else's choice in the future.